app/components/Students.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import store from '../store.jsx'
+import Navbar from './Navbar'
+import { NavLink } from 'react-router-dom'
+
+
+
+const mapStateToProps = function(state) {
+  return {
+    students: state.students
+  }
+}
+
+const mapDispatchToProps = null;
+
+function listStudents(props){
```

Comment:
Earlier I commented that `class` components should have a capitalized first letter.

I think the react convention would be for function components to also be in `PascalCase`.

If you attempted to render this component named in this way I believe the JSX compiler might not understand it.
```js
function lowerCasedComponent () { return <div/>; }

function render () { return <lowerCasedComponent/>; }
```

The parser is looking for upper cased components: `<UpperCasedComponent/>`